metadata-io/src/main/java/com/linkedin/metadata/timeline/differ/SchemaDiffer.java

```diff
@@ -135,25 +135,30 @@ private static ChangeEvent getDescriptionChange(SchemaField baseField, SchemaFie
   // TODO: This could use some cleanup, lots of repeated logic and tenuous conditionals
   private static List<ChangeEvent> computeDiffs(SchemaMetadata baseSchema, SchemaMetadata targetSchema,
       Urn datasetUrn, ChangeCategory changeCategory) {
-    boolean isOrdinalBasedSchema = isSchemaOrdinalBased(targetSchema);
-    if (!isOrdinalBasedSchema) {
-      // Sort the fields by their field path.
-      if (baseSchema != null) {
-        sortFieldsByPath(baseSchema);
-      }
-      sortFieldsByPath(targetSchema);
+    // Sort the fields by their field path.
```

Comment:
No need to check ordinal based schema thing?

Reply:
The ordinal stuff was based on the assumption that we actually apply ordinal logic from SQL based stores. We do not store the ordinal location of fields so it resulted in incorrect treatment of field removals and renames. i.e.:

Schema {
field1 string
field2 string
field3 string
}
 |
V
Remove field2, rename field3 to new_field
 |
V
Schema {
field1 string
new_field string
}
would result in: Field2 renamed to new_field, field3 removed
Even though the underlying data in new_field would be Field2's data

Reply:
With these changes it will either show up as the correct rename if it can be identified (description, tags, or terms match) or as an add/drop which is more intuitive to explain.

Reply:
Got it-ty